README.md

```diff
@@ -208,6 +209,19 @@ JoliCode\Elastically\Client:
             elastically_bulk_size: 100
 ```
 
+You can also use the Symfony HttpClient for all Elastica communications:
+
+```yaml
+JoliCode\Elastically\Transport\HttpClientTransport: ~
```

Comment:
There is no need to configure a default timeout?

Reply:
Autowiring on HttpClientInterface should pick your pre-configured framework Client.